src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
+
+  let sum = 0;
+
+  men.forEach((m) => sum += m.died - m.born);
+
+  return sum/men.length;
```

Comment:
Add spaces before and after `/` to follow the [style guides](https://mate-academy.github.io/style-guides/javascript.html#194set-off-operators-with-spaces).
```suggestion
  return sum / men.length;
```